analyzer/pkg/rpm/rpm.go

```diff
@@ -104,9 +106,13 @@ func (a rpmPkgAnalyzer) parsePkgInfo(rc io.Reader) ([]types.Package, []string, e
 			}
 		}
 
-		files, err := pkg.InstalledFiles()
-		if err != nil {
-			return nil, nil, xerrors.Errorf("unable to get installed files: %w", err)
+		//check if the package is preinstalled on the system
+		files := []string{}
```

Comment:
nil slice is preferred.

```suggestion
		var files []string
```
https://github.com/golang/go/wiki/CodeReviewComments#declaring-empty-slices